src/course-unit/course-xblock/utils.js

```diff
@@ -0,0 +1,10 @@
+import { getConfig } from '@edx/frontend-platform';
+
+const getXBlockHandlerUrl = async (blockId, handlerName) => {
+  const baseUrl = getConfig().STUDIO_BASE_URL;
+
+  return `${baseUrl}/xblock/${blockId}/handler/${handlerName}`;
+};
+
+// eslint-disable-next-line import/prefer-default-export
+export const getHandlerUrl = async (blockId) => getXBlockHandlerUrl(blockId, 'handler_name');
```

Comment:
These two functions maybe we should move to the `api.js`?

Reply:
[question] Do we need to create a wrapper function instead of a single function?

Reply:
Yes, thanks
It makes sence